PpW HC Friday 2 recap (#463)
tpwres/teczki-polskiego-wrestlingu: content/e/ppw/2024-09-20-ppw-hardcore-friday-2.md

* Heels start to beat down Samson during Blade's entrance as he enters the ring; he makes a save and the match is even. Action spills outside the ring, Gryffin and Samson brawl at the bar table. Crowd urges Samson to kill Gryffin.
* Now, Gryffin and Blade fight in the ring, while Samson and Isnorr fight outside. Gryffin tosses Blade outside onto the other two.
* Samson and Gryffin in the ring, Gryffin lays Samson out with a chairshot. He continues to pummel Samson, while the crowd urges Samson to fight back. He makes a comeback, but it's short-lived and Gryffin continues to punish him.
* Samson and Blade fight Isnorr for some near falls, eventually Gryffin makes his return to the ring. Gryffin is laid on the table, Samson with a frog splash for a near fall - Isnorr breaks the pin from the outside.
* Low blow from Isnorr and heels get the upper hand. Gryffin straps Samson to the corner. They slam Blade through a board on the chairs, while Samson is only able to helplessly watch.
* Isnorr pins Blade for a three-count. Winners: Isnorr and Gustav Gryffin.

> * Heels continue to pummel Blade with helpless Samson still in the corner. Olgierd joins other heels in the beatdown now and hits Braindead on Blade. Axel Fox enters for a save to a monster pop. A masked figure lays him out with a kendo stick - it's Mister Z. They all beat Fox down and Biesiad is in, throwing some scaffolding into the ring.

Reviewer: ```suggestion

#### Finale

* Heels continue to pummel Blade with helpless Samson still in the corner. Olgierd joins other heels in the beatdown now and hits Braindead on Blade. Axel Fox enters for a save to a monster pop. A masked figure lays him out with a kendo stick - it's Mister Z. They all beat Fox down and Biesiad is in, throwing some scaffolding into the ring.
```

* Biesiad attempts a pinfall but is reminded that the match can only be won by putting the oponent into the van parked outside the venue. After more action in the ring, they brawl towards the outside and the van.
* They brawl using the van and some railings set up due to area's reconstruction. Biesiad is thrown into the van and Feager closes one side of the door, then the next other... but Biesiad blocked the door with a board. He dives out of the van and makes his comeback on Feager.
* Now Feager is put in the van, Biesiad joins him and locks both of them inside. The referee starts the count and the van drives away to everyone's confusion. Nonplussed, the referee announces that the fight carries on, as the vehicle disappears around the bend.
* As the crowd returns to the venue, Biesiad's music is heard and Michael HT declares Biesiad to be the winner. Winner: apparently Biesiad, unless confirmed otherwise by PpW.

#### Break

#### Marcelito vs Jacob Crane

* Referee of the match: Seweryn.
* [Jacob Crane](@/w/jacob-crane.md) is out first, making a babyface entrance. [Marcelito](@/w/marcelito.md) is next. They start trading technical offense.

> * Crowd starts singing, but it's unrelated to the match - it's now 21:37 (the hour of pope John Paul II's death) and the song is ["Barka"](https://en.wikipedia.org/wiki/Lord,_You_Have_Come_to_the_Lakeshore).

Reviewer: ```suggestion
* Crowd starts singing, but it's unrelated to the match - it's now 21:37 and the song is ["Barka"](https://en.wikipedia.org/wiki/Lord,_You_Have_Come_to_the_Lakeshore).
```

* Heels start to beat down Samson during Blade's entrance as he enters the ring; he makes a save and the match is even. Action spills outside the ring, Gryffin and Samson brawl at the bar table. Crowd urges Samson to kill Gryffin.
* Now, Gryffin and Blade fight in the ring, while Samson and Isnorr fight outside. Gryffin tosses Blade outside onto the other two.
* Samson and Gryffin in the ring, Gryffin lays Samson out with a chairshot. He continues to pummel Samson, while the crowd urges Samson to fight back. He makes a comeback, but it's short-lived and Gryffin continues to punish him.
* Samson and Blade fight Isnorr for some near falls, eventually Gryffin makes his return to the ring. Gryffin is laid on the table, Samson with a frog splash for a near fall - Isnorr breaks the pin from the outside.
* Low blow from Isnorr and heels get the upper hand. Gryffin straps Samson to the corner. They slam Blade through a board on the chairs, while Samson is only able to helplessly watch.
* Isnorr pins Blade for a three-count. Winners: Isnorr and Gustav Gryffin.

#### Finale

> * Heels continue to pummel Blade with helpless Samson still in the corner. Olgierd joins other heels in the beatdown now and hits Braindead on Blade. Axel Fox enters for a save to a monster pop. A masked figure lays him out with a kendo stick - it's Mister Z. They all beat Fox down and Biesiad is in, throwing some scaffolding into the ring.

Reviewer: ```suggestion
* The Heels continue to pummel Blade, with a helpless Samson still in the corner. Olgierd joins the beatdown now and hits Braindead on Blade. Axel Fox enters for a save to monster pop. A masked figure lays him out with a kendo stick - it's Mister Z. They all beat Fox down.
* Outside, the van returns, Biesiad runs into the venue. After a brief brawl, steps outside for a moment,  and returns with a piece of aluminium truss, throwing it into the ring, forcing the heels to take notice.
```

  - '[Marcelito](@/w/marcelito.md)'
- - '[Gustav Gryffin](@/w/gustav-gryffin.md), [Isnorr](@/w/isnorr.md)'
  - '[Samson](@/w/samson.md), [Johnny Blade](@/w/johnny-blade.md)'
  - s: Anarchy Cyclone Tag Team Match
- - '[Axel Fox](@/w/axel-fox.md)'
  - '[Biesiad Strong](@/w/biesiad.md)'
  - '[Gustav Gryffin](@/w/gustav-gryffin.md)'
  - '[Johnny Blade](@/w/johnny-blade.md)'
  - '[Mister Z](@/w/johnny-blade.md)'
  - '[Olgierd](@/w/olgierd.md)'
  - '[Samson](@/w/samson.md)'
  - g: true
    s: Zmowa vs Zmowa przeciwko zmowie face off
- credits:
    Referees: '[Sędzia Karol Górski](@/w/sedzia-karol-gorski.md), [Sędzia Seweryn](@/w/sedzia-seweryn.md)'
    Host and Ring Announcer: '[Michael HT](@/w/michael-ht.md)'
{% end %}

#### Recap
* There was no official transmission, but a [fancam live stream](https://www.youtube.com/watch?v=aHkaPbVlPf8) is available.
* [PpW's host](@/w/michael-ht.md) makes his entrance to ["Koty z Mokotowa"](https://www.youtube.com/watch?v=tnY3qfmS-6k) (not availbable in the unofficial stream, but confirmed by fans in attendance). He runs through the card of the's show and hypes up the crowd. Gustav Gryffin's music interrupts him.

> * [Gustav](@/w/gustav-gryffin.md), [Olgierd](@/w/olgierd.md) and [Isnorr](@/w/isnorr.md) make their entrance. Gustav, wearing a suit, cuts a promo about PpW's growth being due to Zmowa's actions. He lists the members: Olgierd, a prospective member Isnorr, Jakub Linde, [Marco Hammers](@/w/marco-hammers.md) and [Mister Z](@/w/mister-z.md) (named a "true messiah of Polish wrestling" - a shot at ["Pan" Pawłowski](@/w/pan-pawlowski.md). The crowd boos and chants "Mister Zjeb" (roughly _Mister Fuck_). Gustav lists the previous champions, then says the belt is a relic of the past, stomps on it and kicks it out of the ring. Olgierd unzips his sweatshirt to present Gustav with a new, personalised, white [PpW Championship](@/c/ppw-championship.md). He gets "fuck you" and "respect the belt" chants. He dares anyone who has a problem with this to get better and do something about it... but nobody can.

Reviewer: ```suggestion
* [Gustav](@/w/gustav-gryffin.md), [Olgierd](@/w/olgierd.md) and [Isnorr](@/w/isnorr.md) make their entrance. Gustav, wearing a suit, cuts a promo about PpW's growth being due to Zmowa's actions. He lists the members: Olgierd, a prospective member Isnorr, Jakub Linde, [Marco Hammers](@/w/marco-hammers.md) and [Mister Z](@/w/mister-z.md) (named a "true messiah of Polish wrestling" - a shot at ["Pan" Pawłowski](@/w/pan-pawlowski.md). The crowd boos and chants "Mister Zjeb" (roughly _Mister Asshole_). Gustav lists the previous champions, then says the belt is a relic of the past, spits and stomps on it, then kicks it out of the ring. On command, Olgierd unzips his sweatshirt to present Gustav with a new, personalised, white [PpW Championship](@/c/ppw-championship.md). He gets "fuck you" and "respect the belt" chants. He dares anyone who has a problem with this to get better and do something about it... but nobody can.
```

* Michael HT makes his entrance to announce next match.
* [Referee Seweryn](@/w/sedzia-seweryn.md) slides into the ring. Crowd chants for "violence".

> * Out next is [Axel Fox](@/w/axel-fox.md). He makes his entrance and calls out Mister Z or anybody else. Making his entrance is rookie Oskar, wearing all black, Young Lion-style gear. The crowd makes fun of it with "show your ass", "wear some pants" and "show your dick" chants. Fox offers a handshake, but Oskar refuses, establishing himself as a heel in this match. The crowd's clearly on Fox's side. Oskar establishes early dominance with power moves and berating his smaller oponent. Fox makes a comeback, "this is awesome" chants erupt. Fox wins with a top rope elbow drop to a huge pop. He offers a handshake to the rookie, which he now accepts. While Fox celebrates, some female fans want him to show his privates at the backstage.

Reviewer: ```suggestion
* Out next is [Axel Fox](@/w/axel-fox.md). He makes his entrance and calls out Mister Z or anybody else. Making his entrance is rookie Oskar, wearing all black, Young Lion-style gear. The crowd makes fun of it with "show your ass", "wear some pants" and "show your dick" chants. Fox offers a handshake, but Oskar refuses, establishing himself as a heel in this match. The crowd's clearly on Fox's side. Oskar establishes early dominance with power moves and berating his smaller oponent. Fox makes a comeback, "this is awesome" chants erupt. Fox wins with a top rope elbow drop to a huge pop. He offers a handshake to the rookie, which he now accepts. Fox celebrates in the ring.
```

* A break in the show.
* [Marcelito](@/w/marcelito.md) vs [Jacob Crane](@/w/jacob-crane.md), referee of the match: Seweryn. Jacob Crane is out first, making a babyface entrance. Marcelito is next. They start trading technical offense.

> It's now 21:37 (the hour of pope John Paul II's death) and crowd starts to sing ["Barka"](https://en.wikipedia.org/wiki/Lord,_You_Have_Come_to_the_Lakeshore). Marcelito releases himself from Crane's grip and starts his sequence of striking moves. Satellite DDT on Crane but only two-count. Crane back with his offense, Emerald Flowsion for a three-count. Winner: Jacob Crane

Reviewer: ```suggestion
Crowd starts singing, but it's unrelated to the match - it's now 21:37 and the song is ["Barka"](https://en.wikipedia.org/wiki/Lord,_You_Have_Come_to_the_Lakeshore). Marcelito releases himself from Crane's grip and starts his sequence of striking moves. Satellite DDT on Crane but only two-count. Crane back with his offense, Emerald Flowsion for a three-count. Winner: Jacob Crane
```

* Crane grabs a mic for a promo; he appreciates the crowd liking his posterior and poses for the fans. He announces leaving for Japan for next 6 months, thanks the fans and wishes them fun for the rest of the show.

> * Anarchy Cyclone Tag Team Match - [Isnorr](@/w/isnorr.md) & [Gustav Gryffin](@/w/gustav-gryffin.md) vs [Johnny Blade](@/w/johnny-blade.md) & [Samson](@/w/samson.md). Heel team makes their entrances - Isnorr first to moderate heel reaction, then Gustav Gryffin. Samson makes his entrance with a trash can full of weapons to a huge pop. Johnny Blade joins him with a makeshift mallet (water bin on a kendo stick). Heels start to beat down Samson during Blade's entrance as he enters the ring; he makes a save and the match is even. Action spills outside the ring, Gryffin and Samson brawl at the bar table. Crowd urges Samson to kill Gryffin. Now, Gryffin and Blade fight in the ring, while Samson and Isnorr fight outside. Gryffin tosses Blade outside onto the other two. Samson and Gryffin in the ring, Gryffin lays Samson out with a chairshot. He continues to pummel Samson, while the crowd urges Samson to fight back. He makes a comeback but it's shortl-lived and Gryffin continues to punish him. Samson and Blade fight Isnorr for some near falls, eventually Gryffin makes his return to the ring. Gryffin is laid on the table, Samson with a frog splash for a near fall - Isnorr breaks the pin from the outside. Low blow from Isnorr and heels get the upper hand. Gryffin straps Samson to the corner. They slam Blade through a board put on the chairs, while Samson is only able to watch helplessly. Isnorr pins Blade for a three-count. Winners: Isnorr and Gustav Gryffin.

Reviewer: ```suggestion
* Anarchy Cyclone Tag Team Match - [Isnorr](@/w/isnorr.md) & [Gustav Gryffin](@/w/gustav-gryffin.md) vs [Johnny Blade](@/w/johnny-blade.md) & [Samson](@/w/samson.md). Heel team makes their entrances - Isnorr first to moderate heel reaction, then Gustav Gryffin. Samson makes his entrance with a trash can full of weapons to a huge pop. Johnny Blade joins him with a makeshift mallet (water bin on a kendo stick). Heels start to beat down Samson during Blade's entrance as he enters the ring; he makes a save and the match is even. Action spills outside the ring, Gryffin and Samson brawl at the bar table. Crowd urges Samson to kill Gryffin. Now, Gryffin and Blade fight in the ring, while Samson and Isnorr fight outside. Gryffin tosses Blade outside onto the other two. Samson and Gryffin in the ring, Gryffin lays Samson out with a chairshot. He continues to pummel Samson, while the crowd urges Samson to fight back. He makes a comeback, but it's short-lived and Gryffin continues to punish him. Samson and Blade fight Isnorr for some near falls, eventually Gryffin makes his return to the ring. Gryffin is laid on the table, Samson with a frog splash for a near fall - Isnorr breaks the pin from the outside. Low blow from Isnorr and heels get the upper hand. Gryffin straps Samson to the corner. They slam Blade through a board on the chairs, while Samson is only able to helplessly watch. Isnorr pins Blade for a three-count. Winners: Isnorr and Gustav Gryffin.
```
